Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/CWADebug.kt

```diff
@@ -1,8 +1,24 @@
 package de.rki.coronawarnapp.util
 
+import android.app.Application
 import de.rki.coronawarnapp.BuildConfig
+import de.rki.coronawarnapp.util.debug.FileLogger
+import timber.log.Timber
 
 object CWADebug {
```

Comment:
Is this object just meant to be used in the Application Class or does it have other uses as well down the line?

Reply:
This is the only Singleton that we may keep, as debug related code is usually not injected. When the amount of "debug" init code grows smaller, the class should be split up though. For now I think this grouping is okay, what do you think?

Reply:
Agreed, I think right now the amount of Debug-Code that would need to be in here is small in foreseeable future